### PR TITLE
[REF] web, *: OWL conversion of gantt extensions

### PR DESCRIPTION
**Pull request by Polymorphe57**
We bring here some fixes, refactoring that allow us to finalize the OWL conversion of the gantt view extensions done in
the PR: odoo/enterprise#37091.

Task-ID: 3184636

### CODE REVIEW

File: addons/mrp/views/mrp_workorder_views.xml [hunk 1]
```diff
@@ -343,8 +343,8 @@
                         <div class="row g-0">
                             <div class="col">
                                 <ul class="ps-1 mb-0 list-unstyled">
-                                    <li><strong>Start Date: </strong> <t t-out="userTimezoneStartDate.format('L LTS')"/></li>
-                                    <li><strong>Stop Date: </strong> <t t-out="userTimezoneStopDate.format('L LTS')"/></li>
+                                    <li><strong>Start Date: </strong> <t t-out="date_planned_start.toFormat('F')"/></li>
+                                    <li><strong>Stop Date: </strong> <t t-out="date_planned_finished.toFormat('F')"/></li>
```
Comment (aab-odoo) [inline addons/mrp/views/mrp_workorder_views.xml:347]: t-esc here as well (consistency with below) ?

File: addons/mail/__manifest__.py [hunk 2]
```diff
@@ -154,6 +154,7 @@
             # Unused by guests and depends on ViewDialogs, better to remove it instead of pulling the whole view dependency tree
             ('remove', 'mail/static/src/components/composer_suggested_recipient/*'),
             ('remove', 'mail/static/src/components/activity_menu_container/*'),
+            ('remove', 'mail/static/src/components/avatar/*'),
```
Comment (aab-odoo) [inline addons/mail/__manifest__.py:157]: commit add Avatar could have a bit more explanation

Comment (aab-odoo) [reply addons/mail/__manifest__.py:157]: and tests?

File: addons/mrp/views/mrp_workorder_views.xml (hunk 1 above)
Comment (Polymorphe57) [reply addons/mrp/views/mrp_workorder_views.xml]: done

File: addons/mail/__manifest__.py (hunk 2 above)
Comment (Polymorphe57) [reply addons/mail/__manifest__.py:157]: done

**Comment by aab-odoo** [comment]
Good job! One more to go :D

robodoo rebase-merge r+

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**